src/services/weather.js

```diff
@@ -30,6 +34,28 @@ async function fetchWeatherData(city) {
   return mapModelToView(response.data);
 }
 
-function getIconURL(icon) {
+export function getIconURL(icon) {
   return weatherImageURL + icon + imageExtension;
 }
+
+function mapModelForecastToView(model) {
+  return _.map(model, mapModelToView);
+}
+
+export const fetchWeatherForcastForCity = async (city, completation) => {
+  const response = await axios.get('https://api.openweathermap.org/data/2.5/forecast', {
+    params: {
+      q: city,
+      APPID: Config.APPID,
+    },
+  });
+
+  completation(
+    mapModelForecastToView(_.filter(response.data.list, o => o.dt_txt.includes('12:00:00')))
+  );
+};
+
+export function getDayNameFromDate(date) {
```

Comment:
this function doesn't belong here. It has nothing to do with weather. Create `date.js` service and put it there.